src/docs/composing_applications.md

```diff
@@ -3,8 +3,6 @@ title: Build your own IDE
 slug: composing_applications
```

Comment:
You seem to remove the `slug` everywhere else, but not here. Why not? 🙂

Reply:
Thanks! somehow I forgot to remove this.